statics/js/components/inject-form.js

```diff
@@ -36,6 +36,10 @@ Vue.component('inject-form', {
           <label for="inject-count">ICMP Identifier</label>\
           <input id="inject-id" type="number" class="form-control input-sm" v-model="id" min="0" />\
         </div>\
+        <div class="form-group">\
+          <label for="payload-length">Payload length</label>\
```

Comment:
in this the user can't set the custom payload msg. i would like to have both payload length and payload with select option. so the user can select either one of that. what you think?